tests/e2e/paths/search/granules/granules_views_map_interactions.spec.js

```diff
@@ -72,12 +71,6 @@ test.describe('When clicking on a granule on the map', () => {
       })
     })
 
-    await page.route(/colormaps\/VIIRS_NOAA20_Brightness_Temp_BandI5_Day/, async (route) => {
```

Comment:
I didn't really see the value of having this mocked in this test since we have explicit colormaps tests but, if perferred I can put back in